unit_test/test_spdm_requester/get_digests.c

```diff
@@ -1484,6 +1522,58 @@ static void libspdm_test_requester_get_digests_case25(void **state)
 #endif
 }
 
+
+/**
+ * Test 26: a request message is successfully sent and a response message is successfully received
+ * Expected Behavior: requester returns the status LIBSPDM_STATUS_SUCCESS ,
+ **/
```

Comment:
Here the comment does not mention the check point `MULTI_KEY`, we should highlight the topic.